File: Medium/167-Two-Sum-II/Solutions.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        kiri = 0
        kanan = len(numbers)-1
        while kiri < kanan:
            jumlah = numbers[kiri] + numbers[kanan]
            if jumlah == target: return [kiri+1, kanan+1]
            elif jumlah < target: kiri += 1
            else: kanan -= 1
        return []

File: Medium/167-Two-Sum-II/Solutions.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int kiri = 0;
        int kanan = numbers.size()-1;
        while(kiri < kanan){
            int jumlah = numbers[kiri] + numbers[kanan];
            if (jumlah == target) return {kiri+1, kanan+1};
            else if (jumlah < target) kiri++;
            else kanan--;
        }
        return {};
    }
};